guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+            <td>Miguel operates a lightning node so he can receive payments at his business</td>
+            <td>Wagner has a mobile bitcoin wallet that supports lightning</td>
+         </tr>
+         <tr>
+            <td>What's their role in the lightning network?</td>
+            <td>Rachel acts more like a hub for routing other's lightning payments, and is rarely the origin or endpoint</td>
+            <td>Miguel is usually the endpoint for lightning payments</td>
+            <td>Wagner is sometimes the origin and sometimes the endpoint for lightning payments</td>
+         </tr>
+         <tr>
+            <td>Where does it run?</td>
+            <td>Runs on dedicated hardware, like a Raspberry Pi or rack-mounted server</td>
+            <td>Runs on a cloud service provider</td>
+            <td>Runs on Wagner's phone</td>
+         </tr>
+         <tr>
+            <td>When is it online?</td>
+            <td>Always online</td>
+            <td>Always online</td>
+            <td>Only online when Wagner opens the app</td>
+         </tr>
+         <tr>
+            <td>How many lightning channels?</td>
+            <td>Many lightning channels</td>
+            <td>Several lightning channels</td>
+            <td>Few lightning channels</td>
+         </tr>
+         <tr>
+            <td>How much liquidity?</td>
+            <td>Massive liquidity</td>
+            <td>Medium liquidity</td>
+            <td>Less liquidity</td>
+         </tr>
+         <tr>
+            <td>Where does the liquidity come from?</td>
+            <td>Liquidity is self-funded</td>
+            <td>Liquidity is usually sourced externally (from an LSP, for example)</td>
+            <td>Liquidity is usually sourced externally (from an LSP, for example)</td>
+         </tr>
+         <tr>
+            <td>How much do they know about lightning?</td>
+            <td>Rachel is a lightning node operator and is very cognizant of every decision she makes with her node</td>
+            <td>Miguel knows enough about operating a node to make his business function</td>
+            <td>Wagner is not even aware that his phone has a lightning node on it</td>
+         </tr>
+      </tbody>
+   </table>
+</div>
+
+Currently, most of the thinking in the Design Guide is focused on mobile wallet users such as Wagner. However, it is helpful to understand that routing nodes play a very important role in getting user funds across the lightning network.
+
+## Liquidity examples
+
+The following examples are designed to help you understand how liquidity works on the lightning network.
+
+Lori and Lamar, two routing node operators, open a channel together. Lori initiates the channel opening with 1,000,000 sats. This is the amount she makes available for transactions in the channel. Her initial outbound liquidity is the full amount of 1,000,000 sats, and her initial inbound liquidity is 0 sats (since Lamar has not committed any sats to the channel).
```

Comment:
Would be good to mention that channels are usually funded by one party. I made a comment lower on in the page about adding content on cooperative channel opens (previously dual funded channels)

Reply:
Added "further reading" tip box